packages/editor/themes/affine.css

```diff
@@ -10,3 +10,7 @@
 :root {
   --affine-editor-width: 720px;
 }
+
+.dg > ul {
+  overflow: scroll;
+}
```

Comment:
What is the function of this style?

Reply:
It’s css selector of dat.gui menu , to make container scrollable, rather than hide overflowed content by default